src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html

```diff
@@ -37,25 +37,22 @@ <h1>Students</h1>
                   <div class="row text-center text-sm-start" *ngIf="courseTab.stats">
                     <div class="col-md-4">
                       <div class="card">
-                        <div class="card-body" id="num-students">
-                          <h3 class="card-title inline">{{courseTab.stats.numOfStudents}}</h3>
-                          <h4 class="card-text inline"> students</h4>
+                        <div class="card-body" id="num-students" >
+                          <p class="card-title inline"><font size="4"> {{courseTab.stats.numOfStudents}}  students </font></p>
                         </div>
                       </div>
                     </div>
                     <div class="col-md-4">
                       <div class="card">
                         <div class="card-body" id="num-sections">
-                          <h3 class="card-title inline">{{courseTab.stats.numOfSections}}</h3>
-                          <h4 class="card-text inline"> sections</h4>
+                          <p class="card-title inline"> <font size="4">{{courseTab.stats.numOfSections}} sections </font> </p>
```

Comment:
Would it be possible to add a `fw-bold` utility class here? to make the text bold as per before

Also, instead of using the `<font>` tag, would it be possible to use bootstrap utility classes to adjust the font size? (like `fs-5` maybe)

Similar for the other statistics

Reply:
I tried using a class but because of the existing one ("card-title inline"), the other classes won't work. For instance, the words didn't change to bold. Same applies to fs-5, that is why i used the <font> tag so as to avoid putting another class

Reply:
Ah I see, thanks for investigating!

If I'm not wrong, according to the [Bootstrap docs](https://getbootstrap.com/docs/5.0/components/card/#titles-text-and-links), I believe that `card-title` only works on `<h*>` tags. Thus, since we are now using `<p>` tags, the `card-title` does not take affect.

Personally, I don't think it makes sense to have a redundant `card-title` class, since it does not do anything. Thus, we can simply remove the `<p>` tag entirely and add in the `fw-bold fs-5` classes as such:

```
<div class="card-body fw-bold fs-5" id="num-sections">
  {{courseTab.stats.numOfSections}} sections
</div>
```

This will look like this:

![Screen Shot 2023-04-22 at 2 28 46 PM](https://user-images.githubusercontent.com/25262042/233766933-4fe8d6ad-eb7c-4774-9449-a6ea3d4cfec3.png)

This also follows the [Bootstrap template of a simple card with only body text](https://getbootstrap.com/docs/5.0/components/card/#body). 

What do you think? @athakaras

Reply:
Thanks for the help! I will try your suggestion and implement it if it works. I will be closing this PR due to some technical issues with the commits and will open a new PR with the changes implemented if that is okay:)